### ISSUE TITLE
[Feature Request] dns and web-based autodiscovery for mail and other services

### ISSUE THREAD

**Issue by dphi** [Froxlor/Froxlor#730]
# Summary

There are a few dns records that help clients to do auto discovery,eg, for mail, caldav, and carddav. It would be great if Froxlor would make sure that they are in the DNS zones.

For domains with email support:
* adding [rfc6186](https://tools.ietf.org/html/rfc6186) compatible records (This is related to #719 .)
* https://developer.mozilla.org/en-US/docs/Mozilla/Thunderbird/Autoconfiguration defines autoconfig.domain
* https://support.microsoft.com/en-us/help/3211279/outlook-2016-implementation-of-autodiscover defines _autodiscover._tcp.domain in step 10.
* The iOS feature would also be nice to have, see https://github.com/mailcow/mailcow-dockerized/blob/master/data/web/mobileconfig.php for an example.

Adding custom records to multiple domains managed by Froxlor would help too, eg, caldav, carddav see [rfc6764](https://tools.ietf.org/html/rfc6764)

Thanks

**Comment by d00p** [comment]
> It would be great if Froxlor would make sure that they are in the DNS zones.

how about the admin decides whether they should be added and how?

Side note: not very high priority as it's dns only related which are rarely used

**Comment by dphi** [comment]
> > It would be great if Froxlor would make sure that they are in the DNS zones.
> 
> how about the admin decides whether they should be added and how?

Agreed, the admin has to configure which settings should be available for a given customer by default. Also: The customer should be able to override these.

Thanks

**Comment by d00p** [comment]
Customer can overwrite any entry via the dns editor if enabled already
